ag-qs-sched.html

```diff
@@ -1422,8 +1422,8 @@ <h3 class="heading-tertiary">Guided Study Groups:</h3>
 
 
 
-                  <tr class="zoom-session" title="General Chemistry I">
-                    <td>Hosted by: Andrea</td>
+                  <tr class="cancel-session" title="General Chemistry I">
+                    <td>&times CANCELLED 3/17 <br>Hosted by: Andrea</td>
```

Comment:
Please make sure to preview these changes in the live server. Whenever you use an HTML code like &times, you need to end the code with a semi-colon. So it should be `&times;`